systems/robotInterfaces/@Biped/Biped.m

```diff
@@ -98,7 +101,7 @@
 
       [Axy, bxy] = poly2lincon([0, params.max_forward_step, 0, -params.max_forward_step],...
                                [params.min_step_width, params.nom_step_width, params.max_step_width, params.nom_step_width]);
-      [Axz, bxz] = poly2lincon([0, params.nom_forward_step, params.max_forward_step, params.nom_forward_step, 0, -params.max_forward_step], ...
+      [Axz, bxz] = poly2lincon([0, params.max_forward_step, params.max_forward_step, params.max_forward_step, 0, -params.max_forward_step], ...
```

Comment:
It's necessary to get good automated planning for the tilted blocks. I'll need to revisit this logic later when we have a better idea of the controller's  capabilities